### PR TITLE
Fix account slip filename

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR fix the issue #1533

### CODE REVIEW

File: server/controllers/finance/reports/generalLedger/index.js [hunk 1]
```diff
@@ -66,22 +66,23 @@ function renderAccountSlip(req, res, next) {
...
-  }
-
   const data = {};
 
-  return AccountReport.getAccountTransactions(params.account_id, GENERAL_LEDGER_SOURCE)
+  return Accounts.lookupAccount(params.account_id)
+    .then((account) => {
+      _.extend(data, { account });
+
+      options.filename = 'Extrait'.concat('_', account.number, '_', account.label);
```
Comment (jniles) [inline server/controllers/finance/reports/generalLedger/index.js:75]: Do you think this should include a timestamp (or a date)?  That way a user can generate multiple without overriding the previous?

Comment (mbayopanda) [reply server/controllers/finance/reports/generalLedger/index.js:75]: The `ReportManager` add automatically a date at the end of the filename.

csv_export_name

Comment (jniles) [reply server/controllers/finance/reports/generalLedger/index.js:75]: Ah, I didn't know that.  LGTM then.

**Review by sfount — APPROVED** [review]
LGTM - addresses the referenced issue.

**Pull request merged**